From 776e739b1690c7de11e50e2ae2a77d98bd69a3d6 Mon Sep 17 00:00:00 2001
From: Alan Coopersmith <alan.coopersmith@oracle.com>
Date: Wed, 23 May 2012 21:48:59 -0700
Subject: [PATCH] Stop trying to use NULL for Status values

Fixes gcc errors in 64-bit builds:
XpNotifyPdm.c: In function 'XpGetPdmStartParams':
XpNotifyPdm.c:234:10: error: cast from pointer to integer of different size
XpNotifyPdm.c:271:10: error: cast from pointer to integer of different size
XpNotifyPdm.c:286:10: error: cast from pointer to integer of different size

Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
---
 src/XpNotifyPdm.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/XpNotifyPdm.c b/src/XpNotifyPdm.c
index c1ceb8e..579923c 100644
--- a/src/XpNotifyPdm.c
+++ b/src/XpNotifyPdm.c
@@ -231,7 +231,7 @@ XpGetPdmStartParams (
 	/*
 	 * Error - cannot determine or establish a selection_display.
 	 */
-	return( (Status) NULL );
+	return( (Status) 0 );
     }
 
     /*
@@ -268,7 +268,7 @@ XpGetPdmStartParams (
 	    XCloseDisplay( *selection_display );
 	    *selection_display = (Display *) NULL;
 	}
-	return( (Status) NULL );
+	return( (Status) 0 );
     }
 
     status = XmbTextListToTextProperty( *selection_display, list, 6,
@@ -283,7 +283,7 @@ XpGetPdmStartParams (
 	    XCloseDisplay( *selection_display );
 	    *selection_display = (Display *) NULL;
 	}
-	return( (Status) NULL );
+	return( (Status) 0 );
     }
 
     *type              = text_prop.encoding;
-- 
1.8.2.3

